airnow/cmd_tool.py

```diff
@@ -1,15 +1,28 @@
 # -*- coding: utf-8 -*-
 
 import argparse
-from datetime import date
+import logging
 import os
 import sys
+
+from datetime import date
 from pathlib import Path
 
 import airnow
 
-__DEBUG__ = os.environ.get("AIRNOW_DEBUG", False)
-__DEBUG__ = True
+logging.basicConfig(
+    format="-- [%(asctime)s][%(levelname)s][%(name)s] - %(message)s",
+    datefmt="%Y-%m-%d %H:%M:%S",
+)
+
+logger = logging.getLogger(__name__)
+
+__DEBUG__ = not os.environ.get("AIRNOW_DEBUG", False)
```

Comment:
We can move this to a CLI arg if you'd like. Definitely helpful for initial dev. 

```suggestion
__DEBUG__ = os.environ.get("AIRNOW_DEBUG", True)
```